plugins/processors/starlark/testdata/fail.star

```diff
@@ -0,0 +1,12 @@
+# Example of the way to return a custom error thanks to the built-in function fail
+#
```

Comment:
let's add a statement along the lines of "returning an error will drop the current metric."